tests/app/helpers.py

```diff
@@ -243,6 +243,9 @@ def reset_application_company_details_confirmed_decorator(self):
 
 class BaseApplicationTest:
     def setup_method(self, method):
+        self.app_env_var_mock = patch.dict('gds_metrics.os.environ', {'PROMETHEUS_METRICS_PATH': '_metrics'})
+        self.app_env_var_mock.start()
```

Comment:
Does every test need the metics path envvar? I realise you've already done a load of apps this way 😅

Reply:
Remembered why I did it like this now.
When we do it like so:
https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/889

It means that for most of the tests the metrics will be on the wrong endpoint. It makes our tests less representative.

Whereas the way its done here preserves the _metrics location for the whole test suite

Reply:
(The first commit on that pr is an example of the way we shouldn't do it)